recipes/sdl_mixer/all/conanfile.py

```diff
@@ -56,80 +62,84 @@ def config_options(self):
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
-        if self.options.shared:
-            del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("sdl/2.0.20")
+        # SDL_mixer public header includes several public headers of SDL:
+        # SDL_stdinc.h, SDL_rwops.h, SDL_audio.h, SDL_endian.h and SDL_version.h
+        self.requires("sdl/2.26.5", transitive_headers=True)
```

Comment:
This might be a good opportunity to change the sdl version to 2.26.5 in sdl_image, sdl_net, and sdl_ttf to avoid conflicts when using them together.